### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by huddy987**
Fix header and javaDoc comments

### CODE REVIEW

**Review by IsaakC — APPROVED** [review]

**Review by JawadHossain — APPROVED** [review]

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/SetLocationActivity.java [hunk 1]
```diff
@@ -1,10 +1,27 @@
 package com.example.bookwormadventuresdeluxe2;
 
 /**
- * Allows user to pick an exchange location by placing a marker on a map.
```
Comment (JawadHossain) [inline app/src/main/java/com/example/bookwormadventuresdeluxe2/SetLocationActivity.java:4]: I actually changed this file. Did you resolve the conflict?

**Review by JawadHossain — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/bookwormadventuresdeluxe2/SetLocationActivity.java (hunk 1 above)
Comment (JawadHossain) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/SetLocationActivity.java:4]: I updated comments throughout the files SetLocationActvity and ViewLocationActivity. Could you pls pull from master?

Comment (huddy987) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/SetLocationActivity.java:4]: I must have manually resolved this conflict incorrectly

Comment (JawadHossain) [reply app/src/main/java/com/example/bookwormadventuresdeluxe2/SetLocationActivity.java:4]: oh no worries!

**Review by JawadHossain — APPROVED** [review]

**Pull request merged**